tests/whisper_protocol/crypto_wrappers/curve25519.rs

```diff
@@ -54,19 +59,74 @@ pub fn generate_private_key() -> PrivateKey{
     PrivateKey::from_bytes(*private_key)
 }
 
-#[derive(Clone)]
-#[derive(PartialEq)]
 pub struct PublicKey {
```

Comment:
neither solution is nice, but the one that's in there seems a little cleaner. (offset of 1, rather than an unsafe transmute)